### PR TITLE
SAAS-142 API tests for triggering checks execution

### PR DESCRIPTION
**Pull request by artemgavrilov**
FB: https://github.com/Percona-Lab/pmm-submodules/pull/857

### CODE REVIEW

**Comment by artemgavrilov** [comment]
Assigned wrong person last time)

File: server/checks_test.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		assert.NotNil(t, resp)
+	})
+
+	t.Run("StartSecurityChecksWithDisabledSTT", func(t *testing.T) {
+		defer restoreSettingsDefaults(t)
+		// Disabled STT
+		res, err := client.ChangeSettings(&server.ChangeSettingsParams{
+			Body: server.ChangeSettingsBody{
+				DisableStt:      true,
+				EnableTelemetry: true,
```
Comment (BupycHuk) [inline server/checks_test.go:45]: Is it required field?

Comment (yurkovychv) [reply server/checks_test.go:45]: That is done for readability. The field is not required. Should i remove it?

**Comment by AlekSi** [comment]
Waiting for https://github.com/Percona-Lab/pmm-submodules/pull/857 to finish

**Pull request merged**